dtfs-central-api/src/v1/controllers/utilisation-report-service/fee-record-correction/put-fee-record-correction-transient-form-data.controller/helpers/get-record-correction-transient-form-validation-error.test.ts

```diff
@@ -0,0 +1,161 @@
+import { RECORD_CORRECTION_REASON, MAX_RECORD_CORRECTION_ADDITIONAL_INFO_CHARACTER_COUNT } from '@ukef/dtfs2-common';
+import { TfmFacilitiesRepo } from '../../../../../../repositories/tfm-facilities-repo';
+import { getAdditionalCommentsValidationError, getFacilityIdValidationError } from './get-record-correction-transient-form-validation-error';
+
+describe('get-record-correction-transient-form-validation-error', () => {
+  describe('getAdditionalCommentsValidationError', () => {
+    describe('when "additionalComments" is undefined', () => {
+      const reasons = [RECORD_CORRECTION_REASON.OTHER];
+      const additionalComments = undefined;
+
+      it('should return error message', () => {
+        // Act
+        const errorMessage = getAdditionalCommentsValidationError(reasons, additionalComments);
+
+        // Assert
+        const expectedErrorMessage = 'You must enter a comment';
+
+        expect(errorMessage).toEqual(expectedErrorMessage);
+      });
+    });
+
+    describe('when "additionalComments" is an empty string', () => {
+      const reasons = [RECORD_CORRECTION_REASON.OTHER];
+      const additionalComments = '';
+
+      it('should return error message', () => {
+        // Act
+        const errorMessage = getAdditionalCommentsValidationError(reasons, additionalComments);
+
+        // Assert
+        const expectedErrorMessage = 'You must enter a comment';
```

Comment:
can all the error messages be kept at the top then the repeated once aren't repeated